android/src/main/java/com/swmansion/gesturehandler/core/PanGestureHandler.kt

```diff
@@ -45,6 +45,7 @@ class PanGestureHandler(context: Context?) : GestureHandler<PanGestureHandler>()
   private var activateAfterLongPress = DEFAULT_ACTIVATE_AFTER_LONG_PRESS
   private val activateDelayed = Runnable { activate() }
   private var handler: Handler? = null
+  var stylusData: StylusData = StylusData(0.0, 0.0, 0.0, 0.0, -1.0)
```

Comment:
```suggestion
  var stylusData: StylusData = StylusData(0.0, 0.0, 0.0, 0.0, -1.0)
    private set
```

And the same deal with numbers

Reply:
Done in [b8203f2](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/b8203f29171ad5ff49d1ce0b4e1dfa0a6b2f717c) and [4034348](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/40343489095e239b3920e931b22c24c5879115c8)